internal/profile/default-profile.json

```diff
@@ -95,9 +95,14 @@
 		  ]
 		},
 		"validations": {
-		  "pattern": {
-			"pattern": "^([\\p{Lu}\\p{Ll}][\\p{Lu}\\p{Ll} /\\\\\\.',-]{0,127})$"
-		  }
+			"pattern": {
```

Comment:
why do we change the regexp ? I could imagine that we let the original regexp and we  just add the length fields

Reply:
We don't want a message telling that the pattern is invalid and we can't know in which order patterns will be evaluated